scripts/jenkins/init-node.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Install nvm for this shell
+source ~/.nvm/nvm.sh
+
+# Use version of node.js found in .nvmrc
+nvm install
+
+npm install -g yarn
```

Comment:
[jq](https://stedolan.github.io/jq/) is a good tool for that. You can play with it at https://jqplay.org/